privacyidea/migrations/versions/056b6642ff5d_.py

```diff
@@ -0,0 +1,29 @@
+"""Increase audit signature column size to support 4096-bit RSA keys
```

Comment:
Please add `v3.13: ` in front of the string to make it more distinguishable when showing the migration history.